test/util/validation.js

```diff
@@ -124,6 +124,54 @@ describe("#isIn()", function() {
     });
 });
 
+describe("#isUpper()", function() {
+    it("should verify basic is simple conditions", () => {
+        let result;
+
+        result = yonius.isUpper()("HELLO");
+        assert.strictEqual(result, true);
+
+        result = yonius.isUpper()("HELLO.WORLD");
+        assert.strictEqual(result, true);
+
+        result = yonius.isUpper()(null);
+        assert.strictEqual(result, true);
+
+        assert.throws(
+            () => yonius.isUpper()("illegal!"),
+            err => {
+                assert.strictEqual(err instanceof yonius.ValidationError, true);
+                assert.strictEqual(err.message, "Value contains lower cased characters");
+                return true;
+            }
+        );
+    });
+});
+
+describe("#isLower()", function() {
+    it("should verify basic is simple conditions", () => {
+        let result;
+
+        result = yonius.isLower()("hello");
+        assert.strictEqual(result, true);
+
+        result = yonius.isLower()("hello.hello");
+        assert.strictEqual(result, true);
+
+        result = yonius.isLower()(null);
+        assert.strictEqual(result, true);
+
+        assert.throws(
+            () => yonius.isLower()("ILLEGAL!"),

```

Comment:
please keep using the typical hello and hello world string "HELLO" example

Reply:
Taken from https://github.com/hivesolutions/yonius/blob/35e9dcb87477180b2d3061f19d5075f068b57cca/test/util/validation.js#L141